README.md

```diff
@@ -258,15 +258,12 @@ loss_fn = ...
 for epoch in range(num_epoch):
     train_data.reset()
     for nbatch, batch in enumerate(train_data, start=1):
-        data = gluon.utils.split_and_load(batch.data[0], ctx_list=[context],
-                                          batch_axis=0)
-        label = gluon.utils.split_and_load(batch.label[0], ctx_list=[context],
-                                           batch_axis=0)
+        data = batch.data[0].as_in_context(context)
```

Comment:
Could you please apply the same change to the [imagenet example](https://github.com/horovod/horovod/blob/master/examples/mxnet_imagenet_resnet50.py#L339)?

Reply:
will do